equinor/EchoComponents
# chore: security updates and version bumping

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description

Regular version bumping in the scope of security updates.
Notable updates:

- updated echo-eds to 0.14.3. which broke timepicker component. Made adjustments to it.
- - When user clicks on the timepicker, the dropdown is just displayed for a brief moment
- - Newly added "Clear" button to the Single select ruined the UX. Adjusted already existing CSS to hide the clear button. In the case of TimePicker it's not relevant.
- snapshot tests have been updated. Only className changes were updated, since the new version of EDS made some changes there.
- few, css-loader related dependencies were not upgraded, as thier mayor version update brakes storybook. Didn't manage to make storybook work with the new versions. Would need more time for that.

Before updates:
![image](https://user-images.githubusercontent.com/13872908/142390592-4b737429-b666-4c26-944a-9b2d428b0f1e.png)

After updates:
![image](https://user-images.githubusercontent.com/13872908/142390172-900df71c-4066-4a07-81b5-4257204fba6f.png)

How to test:
- link EchoComponents with yalc to EchopediaWeb
- click around, check components using EDS SingleSelect, or Date time picker is still working as expected
- run EchoComponents storybook with `npm run storybook `, click around, check the different components: they should work as expected
